src/style-guide.scrbl

```diff
@@ -188,6 +190,43 @@ fun sqrt(n :: Number%(non-negative)) -> Number:
 end
 }
 
+@subsection{Parametric Annotations}
+
+You should avoid using @tt{Any} as the subtype of a parametric data structure,
+like a @tt{List}.
+
+In general, a @tt{List<Any>} definition is often undesirable because it
+indicates elements of any type can be members of the list. In particular,
+this subtly implies that multiple elements of different types could exist.
+In other words, @tt{[list: 1, "apple", true]} would satisfy the annotation
+@tt{List<Any>}. Creating @tt{List}s like this is usually bad practice.
+
+In Pyret, we can do this by parametrizing the type. Rather than:
```

Comment:
parameterizing? According to https://en.wikipedia.org/wiki/Parametric_polymorphism:

> We say that the type of `append` is parameterized by `a` for all values of `a`